triehash/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "triehash"
-version = "0.8.4"
+version = "0.9.0"
```

Comment:
this is not required, the bump is in dev-dep

Reply:
ah my bad indeed
```suggestion
version = "0.8.4"
```